src/plugins/discover/public/components/json_code_editor/json_code_editor_common.tsx

```diff
@@ -42,7 +42,7 @@ export const JsonCodeEditorCommon = ({
     return null;
   }
   const codeEditor = (
-    <CodeEditor
+    <CodeEditor // TODO theme needs to be provided in the top level context, in src/plugins/discover/public/build_services.ts, theme is not providing a theme$ observable
```

Comment:
JSON views work on Discover, including Surrounding documents and Single document pages. The TODO can be removed.

<img width="500" alt="Screenshot 2023-06-28 at 10 14 18" src="https://github.com/elastic/kibana/assets/1415710/079f902e-44f3-43ff-a28a-f3b03de2cc0d">
<img width="500" alt="Screenshot 2023-06-28 at 10 13 20" src="https://github.com/elastic/kibana/assets/1415710/a5858a30-e580-4342-abfa-6a658357c7d1">